From b6279ab92bbe632e7b4acaccd7af0e64d7e4fad0 Mon Sep 17 00:00:00 2001
From: Yevgany Tyurin <evgturin@gmail.com>
Date: Sun, 27 Mar 2022 22:45:51 +0300
Subject: [PATCH] 1 update firstFile.txt

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index d15112c..414ae74 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-a
-a
+p
+p
 a
 a
 a
-- 
2.35.1.windows.2

